contracts/monitoring/src/error.rs

```diff
@@ -0,0 +1,15 @@
+use axelar_wasm_std_derive::IntoContractError;
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq, IntoContractError)]
+pub enum ContractError {
+    #[error(transparent)]
+    Std(#[from] StdError),
+
+    #[error("unauthorized")]
```

Comment:
can you add more detail to this error string? See `Unauthorized` errors in other contracts